v19.2/import-into.md

```diff
@@ -184,23 +182,18 @@ Google Cloud:
 ## Known limitations
```

Comment:
probably should add "the table's configured GC TTL must be longer than the IMPORT will take to complete"